lms/templates/lti.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
```

Comment:
Reminder to search for expressions in all templates.  I have not done that.  I just happened to look in this one.  If you find any that seem riskier, feel free to move to a separate PR.

I noticed that this file has a note that one expression was sanitized with bleach, which means it contains some markup and HTML-escaping should be disabled.  See https://github.com/edx/edx-platform/blob/835a857757fb996b29cf59b4ea8efc0813b95dbc/lms/templates/lti.html#L64-L65

Reply:
Done